frontend/components/icons/Disable.tsx

```diff
@@ -5,7 +5,7 @@ interface IDisable {
   color?: Colors;
 }
 
-const Disable = ({ color = "core-fleet-blue" }: IDisable) => {
+const Disable = ({ color = "ui-fleet-black-50" }: IDisable) => {
```

Comment:
Confirmed this default wasn't being used anywhere, only red error color, so okay to switch default.